src/Graph.elm

```diff
@@ -730,39 +697,28 @@ type AcyclicGraph n e
     = AcyclicGraph (Graph n e) (List NodeId)
 
 
-unsafeGet : String -> NodeId -> Graph n e -> NodeContext n e
-unsafeGet msg id graph =
```

Comment:
This just doesn't work any more, so I made the calling code deal with it.

Reply:
It's probably the best to instead call `crashLoop` (see below) with `msg`. Many of the crashes here have no real recovery case and it's better to loop endlessly than to return wrong results in the unlikely case that the invariants here are hurt.